### PR TITLE
DAH-1331/hyperlink-dark-bg

### PR DESCRIPTION
**Pull request by kramduckner**
[DAH-1331](https://sfgovdt.jira.com/browse/DAH-1331)

### CODE REVIEW

**Review by christine-sfg — APPROVED** [review]

File: app/javascript/modules/listingDetails/ListingDetailsAdditionalInformation.tsx [hunk 1]
```diff
@@ -87,7 +105,11 @@ export const ListingDetailsAdditionalInformation = ({
                 <div className={"flex mt-4"}>
                   <span className={"font-bold mr-1"}>{t("listings.realtorCommissionHowTo")}</span>
                   <span>
```
Comment (christine-sfg) [inline app/javascript/modules/listingDetails/ListingDetailsAdditionalInformation.tsx:107]: I know this is out of the scope for this story, but if it's a quick fix can we include it? 
image

Comment (kramduckner) [reply app/javascript/modules/listingDetails/ListingDetailsAdditionalInformation.tsx:107]: > I know this is out of the scope for this story, but if it's a quick fix can we include it? image

@christine-sfg Got a fix in! Mind giving it a look before I merge?

Comment (christine-sfg) [reply app/javascript/modules/listingDetails/ListingDetailsAdditionalInformation.tsx:107]: looks great! thanks for squeezing this in!

File: app/javascript/components/TextTruncate.tsx [hunk 2]
```diff
@@ -10,21 +10,33 @@ import { renderInlineMarkup } from "../util/languageUtil"
...
+  let buttonClassNames = ["button-toggle ml-1"]
+
+  if (buttonClassName) {
+    buttonClassNames = [...buttonClassNames, buttonClassName]
+  }
 
   return (
-    <div className="text-sm">
+    <span className={wrapperClassNames.join(" ")}>
       {isExpanded ? untruncatedText : truncatedText}
```
Comment (christine-sfg) [inline app/javascript/components/TextTruncate.tsx:36]: is this ok to be a span? any concerns about invalid markup?

Comment (kramduckner) [reply app/javascript/components/TextTruncate.tsx:36]: @christine-sfg Changed it to a div and added an inline display!

**Pull request merged**